files/audit-policy.yaml

```diff
@@ -0,0 +1,143 @@
+# These rules override the generic ones in roles.
```

Comment:
Can we templatize these policies? In prod, we'd have to add/remove some rules - for other cluster services, CRDs etc.

Reply:
Added a default policy to not audit anything. The template rules would be overridden for each cluster as needed from our core repo.